pkg/crc/api/api.go

```diff
@@ -9,17 +9,17 @@ import (
 	"github.com/code-ready/crc/pkg/crc/logging"
 )
 
-func CreateAPIServer(socketPath string, config newConfigFunc, machine newMachineFunc) (CrcAPIServer, error) {
+func CreateServer(socketPath string, config newConfigFunc, machine newMachineFunc) (Server, error) {
 	listener, err := net.Listen("unix", socketPath)
 	if err != nil {
 		logging.Error("Failed to create socket: ", err.Error())
-		return CrcAPIServer{}, err
+		return Server{}, err
 	}
 	return createAPIServerWithListener(listener, config, machine)
 }
 
-func createAPIServerWithListener(listener net.Listener, config newConfigFunc, machine newMachineFunc) (CrcAPIServer, error) {
-	apiServer := CrcAPIServer{
+func createAPIServerWithListener(listener net.Listener, config newConfigFunc, machine newMachineFunc) (Server, error) {
```

Comment:
did you want to keep this name or should it be `createServerWithListener`  or doesn't matter since it is not exported?

Reply:
moved to the previous commit, should be better.